service/topology_coordinator.cc

```diff
@@ -848,8 +847,8 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                     break;
                 } catch (group0_concurrent_modification&) {
```

Comment:
is this catch even needed now? I think logging of this should be handled in `handle_topology_coordinator_error` instead.

Reply:
I agree. `handle_topology_coordinator_error` should print a warning on `group0_concurrent_modification`.

Reply:
OK, added logging of `group0_concurrent_modification` to `handle_topology_coordinator_error`. Adjusted the test to wait for the new message.